src/test/java/runge/wordledictionary/WordleGameTest.java

```diff
@@ -0,0 +1,49 @@
+package runge.wordledictionary;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.CharArrayReader;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class WordleGameTest
+{
+    @Test
+    public void WordleGameCorrectTest() throws IOException
+    {
+
+        //given
+        WordleGame game1 = new WordleGame();
+
+        //when
+        String correctWord = game1.getRandomWord();
+        CharResult[] correctArray = {CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct, CharResult.Correct};
+        CharResult[] status = game1.guess(correctWord);
+
+        //then
+        assertArrayEquals(correctArray, status);
+
+    }
+
+    @Test
+    public void WordleGameIncorrectTest() throws IOException
+    {
+        //given
+        WordleGame game1 = new WordleGame();
+
+        //when
+        CharResult[] notCorrect = game1.guess("xxxxx");
+        CharResult[] correctArray = {CharResult.Correct, CharResult.Correct,
+                CharResult.Correct, CharResult.Correct, CharResult.Correct};
+        boolean flag = false;
+        if (notCorrect == correctArray)
+        {
+            flag = true;
+        }
+
+        //then
+        assertFalse(false, String.valueOf(flag));
```

Comment:
https://www.educative.io/answers/how-to-check-if-two-arrays-contain-the-same-elements-in-java